apps/artwork/components/artists/helpers.coffee

```diff
@@ -19,7 +19,7 @@ module.exports =
     has: (section) ->
       switch section
         when 'biography'
-          artist.blurb || artist.bio || artist.biography_blurb
+          artist.blurb || artist.bio || artist.biography_blurb?.text != ""
```

Comment:
ah. I suppose our default value for those (in gravity) is `''` instead of blank/non-existent. Weiiird